apps/web/src/views/Farms/components/FarmTable/Row.tsx

```diff
@@ -109,6 +110,16 @@ const Row: React.FunctionComponent<React.PropsWithChildren<RowPropsWithLoading>>
   const shouldRenderChild = useDelayedUnmount(actionPanelExpanded, 300)
   const { t } = useTranslation()
 
+  const { tokenIds } = useUserBoostedPoolsTokenId()
+
+  const isBoosted = useMemo(() => {
+    if (props.type !== 'v3') return false
+    const tokenIdMap = props.details.stakedPositions
+      .map((d) => d.tokenId.toString())
+      .reduce((a, key) => Object.assign(a, { [key]: true }), {})
+    return Boolean(tokenIds.some((tokenId) => tokenIdMap[tokenId] === true))
```

Comment:
can we just use `reduce` and remove `map`, so that this loop will only run once.

e.g.
```ts
   const tokenIdMap = props.details.stakedPositions
      .reduce((acc, curr) => Object.assign(curr, { [acc.tokenId.toString()]: true }), {})
```

Reply:
got it, thanks